cluster/common.sh

```diff
@@ -179,13 +179,13 @@ function get-kubeconfig-bearertoken() {
 function set_binary_version() {
   if [[ "${1}" == "latest_stable" ]]; then
     KUBE_VERSION=$(gsutil cat gs://kubernetes-release/release/stable.txt)
-    echo "Using latest stable version: ${binary_version}"
+    echo "Using latest stable version: ${KUBE_VERSION}"
```

Comment:
Please also update the function comment to say "KUBE_VERSION" instead of "binary_version"